src/Gemfile

```diff
@@ -1,7 +1,8 @@
 source 'https://rubygems.org'
 
 # You may use http://rbenv.org/ or https://rvm.io/ to install and use this version
-ruby ">= 3.3.2"
+ruby ">= 2.6.10"
```

Comment:
why did you downgrade it? put 3.3.2 back

Reply:
I changed to this version because when creating a new react native 0.75.2 project this version is used.